internal/clients/team3/client.go

```diff
@@ -62,6 +62,8 @@ type client struct {
 	params islandParams
 	// iigoInfo caches information regarding iigo in the current turn
 	iigoInfo iigoCommunicationInfo
+	// last sanction score cache to determine wheter or not we have been caugth in the last turn
+	last_sanctions roles.IIGOSanctionScore
```

Comment:
This value isn't set anywhere. You should set this in `resetIIGOInfo` above the line that resets the sanctions struct (which I'm about to add whoops)